code/__DEFINES/construction/structures.dm

```diff
@@ -62,7 +62,14 @@
 #define GEAR_SECURE 1
 #define GEAR_LOOSE 2
 
-// Stationary gas tanks
+//Stationary gas tanks
 #define TANK_FRAME 0
 #define TANK_PLATING_UNSECURED 1
 
+//Computer frames construction states
+#define COMPUTER_FRAME_UNANCHORED 0
+#define COMPUTER_FRAME_ANCHORED 1
```

Comment:
You shouldn't need a state for unanchored and anchored, that's already tracked by whether the computer frame is... anchored or unanchored